packages/devtools_app/lib/src/inspector/flutter/story_of_your_layout/arrow.dart

```diff
@@ -0,0 +1,280 @@
+// Copyright 2019 The Chromium Authors. All rights reserved.
+// Use of this source code is governed by a BSD-style license that can be
+// found in the LICENSE file.
+
+import 'package:flutter/material.dart';
+import 'package:flutter/rendering.dart';
+import 'package:flutter/widgets.dart';
+
+enum ArrowType { up, left, right, down }
+
+/// Widget that wraps and center another widget horizontally with bidirectional arrow (left & right)
+@immutable
+class BidirectionalHorizontalArrowWrapper extends StatelessWidget {
+  const BidirectionalHorizontalArrowWrapper({
+    Key key,
+    this.arrowColor = Colors.white,
+    this.arrowHeadSize = 16.0,
+    this.arrowStrokeWidth = 2.0,
+    @required this.child,
+  })  : assert(child != null),
+        super(key: key);
+
+  final Color arrowColor;
+  final double arrowHeadSize;
+  final double arrowStrokeWidth;
+  final Widget child;
+
+  @override
+  Widget build(BuildContext context) {
+    return Row(
+      mainAxisSize: MainAxisSize.min,
+      mainAxisAlignment: MainAxisAlignment.center,
```

Comment:
nit: are these the defaults? If so, remove.

Reply:
the MainAxisAlignment default is start, but you are right for the CrossAxisAlignment it's default to center